lib/messages/blob.js

```diff
@@ -19,10 +20,22 @@ class Blob extends Message {
   }
 
   getCore() {
-    let lines = Buffer.from(this.blob.content, 'base64').toString().split(/\n/);
+    const buffer = Buffer.from(this.blob.content, 'base64');
+
+    let lines = buffer.toString().split(/\n/);
```

Comment:
Could this be moved after the binary check to avoid unnecessary string manipulation?